src/common/containers/builds.js

```diff
@@ -25,7 +36,7 @@ class Builds extends Component {
         <h1>{fullName} builds</h1>
         <BuildHistory account={account} repo={repo}/>
         { this.props.isFetching &&
-          <span>Loading...</span>
+          <div className={styles.spinner}><Spinner /></div>
```

Comment:
Not sure if we want to keep spinner visible every time we fetch (even if list of builds is already displayed), but at least now it's a nice way to show that auto refresh works (and something is going on in background).